app/presenters/document/web_resource_presenter.rb

```diff
@@ -218,17 +218,6 @@ def media_metadata
       }
     end
 
-    def attribution_snippet(format)
-      # Disabled as textAttributionSnippet and htmlAttributionSnippet have malformed URLs
-      # field_value("#{format}"AttributionSnippet")
-
-      # Temporary workaround until textAttributionSnippet and htmlAttributionSnippet
-      # are fixed upstream
-      # TODO: Remove this workaround
-      snippet = field_value("#{format}AttributionSnippet")
-      snippet&.gsub(record_presenter.field_value('europeanaAggregation.edmLandingPage'), record_presenter.edm_landing_page)
```

Comment:
we can remove 'record_presenter' from the attr_reader decelerations on line 16 then